### PR TITLE
Fix icon, add translate, add Czech language

### PR DESCRIPTION
**Pull request by phavel**
This is proposal for improvements.

### CODE REVIEW

File: syntax.php [hunk 1]
```diff
@@ -184,7 +184,7 @@ function render($format, Doku_Renderer $renderer, $data) {
         $data += $this->query($data['id']);
 
         // link title
-        $link = 'issue #'. $data['id'];
+        $link = $this->getLang('issue') . ' #'. $data['id'];
```
Comment (glensc) [inline syntax.php]: basic translation principles, word order in sentence not the same in all language.

must include issue id in translation as well, and use `sprintf %s`.

File: style.css [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .iw_eventum {
-	background: transparent url(images/eventum.gif) 0px 1px no-repeat; padding-left: 16px;
+	background: transparent url(images/eventum.gif) 0px 0px no-repeat; padding-left: 20px;
```
Comment (glensc) [inline style.css:2]: what's this change? attach screenshot of before and after of the css change please.

**Comment by glensc** [comment]
and you should had created separate PR for each change. as currently i can't merge any of them as some of the changes have issues.

File: style.css (hunk 2 above)
Comment (phavel) [reply style.css:2]: differences:

before:
plugin-edit-before

after:
plugin-edit-after

Test on browser Chrome 58.

File: syntax.php (hunk 1 above)
Comment (phavel) [reply syntax.php]: Is create fix commit OK now? Or should I create new commits and new pull request?

**Comment by glensc** [comment]
did not notice your changes after review. merged now.

**Pull request merged**